data/brands/amenity/fast_food.json

```diff
@@ -1210,6 +1210,20 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Crosstown",
+      "locationSet": {"include": ["gb"]},
+      "matchNames": ["crosstown doughnuts", "crosstown donuts"],
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Crosstown",
+        "brand:wikidata": "Q109943342",
+        "cuisine": "donut",
+        "diet:vegan": "yes",
```

Comment:
In #5727 , `diet:vegan=yes` should be consider whether can be merged into main branch.

You can have a look at that PR

Reply:
I added this tag as they state on their website that they produce a selection of vegan doughnuts, which is a key part of their brand https://www.crosstown.co.uk/vegan-doughnuts/

Reply:
I looked at the website and I agree, the vegan offerings are a key part of their brand and so the tag is ok 👍